src/main/java/uk/gov/hmcts/reform/roleassignment/domain/service/createroles/CreateRoleAssignmentService.java

```diff
@@ -295,54 +297,28 @@ private void insertRequestedRole(AssignmentRequest assignmentRequest,
                     "Requested Role has been rejected due to following new/existing assignment Ids :"
                         + rejectedAssignmentIds.toString());
             }
-            requestedAssignment.setStatus(status);
-            // persist history in db
-            HistoryEntity entity = persistenceService.persistHistory(
-                requestedAssignment,
-                assignmentRequest.getRequest()
-            );
-            requestedAssignment.setId(entity.getId());
-            requestEntity.getHistoryEntities().add(entity);
-        }
-        //Persist request to update relationship with history entities
-        persistenceService.updateRequest(requestEntity);
-        logger.info(String.format(
-            "insertRequestedRole execution finished at %s . Time taken = %s milliseconds",
-            System.currentTimeMillis(),
-            System.currentTimeMillis() - startTime
-        ));
-    }
-
-    private void rejectPreviouslyApprovedAssignments(AssignmentRequest assignmentRequest,
-                                                     Status status,
-                                                     List<UUID> rejectedAssignmentIds) {
-        for (RoleAssignment requestedAssignment : assignmentRequest.getRequestedRoles()) {
-            if (!rejectedAssignmentIds.isEmpty()
-                && (status.equals(Status.REJECTED) || status.equals(Status.DELETE_REJECTED))
-                &&
-                (requestedAssignment.getStatus().equals(Status.APPROVED)
-                    || requestedAssignment.getStatus().equals(Status.CREATED)
-                    || requestedAssignment.getStatus().equals(Status.DELETE_APPROVED))) {
-                requestedAssignment.setLog(
-                    "Requested Role has been rejected due to following new/existing assignment Ids :"
-                        + rejectedAssignmentIds.toString());
-            }
-
             if (requestedAssignment.getStatus() == Status.APPROVED
-                || requestedAssignment.getStatus() == Status.DELETE_APPROVED) {
+                || requestedAssignment.getStatus() == Status.DELETE_APPROVED || requestedAssignment.getStatus().equals(
+                Status.CREATED)) {
```

Comment:
Ir seems we don't need a separate if block for these instruction, it should be added in the above if block.